showcase/neic_pari.md

```diff
@@ -0,0 +1,351 @@
+: Free text
+
+Additional fields
+: Extend this table with additional rows corresponding to fields from the [NeIC PaRI Data dictionary for SARS-CoV-2 genome data](https://docs.google.com/spreadsheets/d/1gNpdZKOUKPemMUHR107JRSeaWjPczlMIUp5fP5-kR9g/edit#gid=1524315810)
+
+Custom fields
+: Extend this table with additional rows corresponding to additional factors documented for this experiment
+
+
+### Genome assembly
+
+Description
+: A free-form description of the consensus sequence and assembly method / configuration
+: Free text
+
+Additional fields
+: Extend this table with additional rows corresponding to fields from the [NeIC PaRI Data dictionary for SARS-CoV-2 genome data](https://docs.google.com/spreadsheets/d/1gNpdZKOUKPemMUHR107JRSeaWjPczlMIUp5fP5-kR9g/edit#gid=1524315810)
+
+Custom fields
+: Extend this table with additional rows corresponding to additional factors documented for this assembly
+
+## Selected references on data sharing
+
+The following resources can be useful when planning a new study or revising practices for documentation or data management:
+
+-   [PaRI Data Dictionaries and Data Mappings](https://docs.google.com/spreadsheets/d/1gNpdZKOUKPemMUHR107JRSeaWjPczlMIUp5fP5-kR9g/edit#gid=1524315810)
+-   [ELIXIR’s RDMkit](https://rdmkit.elixir-europe.org/index.html)
+-   [EBI: SARS-CoV-2 (Severe acute respiratory syndrome coronavirus 2) Submissions — ena-browser-docs latest documentation](https://ena-browser-docs.readthedocs.io/en/latest/help_and_guides/sars-cov-2-submissions.html)
+-   [GISAID: Submitting Data to the EpiFlu™ Database](https://www.gisaid.org/epiflu-applications/submitting-data-to-epiflutm/)
+-   [Galaxy training: Submitting raw sequencing reads to ENA](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/upload-data-to-ena/tutorial.html)
+-   [The PHA4GE SARS-CoV-2 Contextual Data Specification for Open Genomic Epidemiology](https://www.preprints.org/manuscript/202008.0220/v1).
+-   [SARS-CoV-2 ENA submission workflow + guidance for structuring and releasing metadata](https://dx.doi.org/10.17504/protocols.io.buqnnvve).
+-   [SARS-CoV2 GISAID submission protocol](https://dx.doi.org/10.17504/protocols.io.bumknu4w).
+
+## National support / infrastructures
+
+### Estonia
+
+- [https://covid19dataportal.ee/](https://covid19dataportal.ee/) 
+- [https://koroona.ut.ee/](https://koroona.ut.ee/)
+
+### Norway
+
+- [ELIXIR Norway Research support](https://elixir.no/helpdesk)
+- [ELIXIR Norway COVID-19 resources](https://elixir.no/corona)
+- [ELIXIR Norway SARS-Cov-2 database](https://covid19.sfb.uit.no/)
+
+### Sweden
+
+- [Swedish COVID-19 Data Portal Support Services](https://www.covid19dataportal.se/support_services/)
+- [SciLifeLab Data Guidelines](https://scilifelab-data-guidelines.readthedocs.io/en/latest/docs/covid-19/index.html)
+
+### NeIC PaRI case study
```

Comment:
6 cont. SPLIT SUGGESTION Perhaps the 'case study' could also integrate into national pages?